rules/windows/defense_evasion_enable_inbound_rdp_with_netsh.toml

```
@@ -14,6 +14,43 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Remote Desktop Enabled in Windows Firewall"
```

```suggestion
name = "Remote Desktop Enabled in Windows Firewall by Netsh"
```

Thinking about this from a telemetry perspective if aggregating data by rule name, "Netsh" would potentially give more of an understanding.